index.html

```diff
@@ -107,35 +114,35 @@ <h2 class="txt-center my-2">Featured Categories</h2>
             <span class="categogry__name txt-center txt-bold txt-2xl"
               >DSLR</span
             >
-          </div>
-          <div class="category__card">
+          </a>
+          <a href="/ProductListing/productListing.html" class="category__card">
             <img
               src="/Assets/tripods.jpg"
               class="img-responsive"
-              alt="camera"
+              alt="tripod"
               loading="lazy"
             />
             <span class="categogry__name txt-center txt-bold txt-2xl"
               >Tripods</span
             >
-          </div>
-          <div class="category__card">
-            <img src="/Assets/lens.jpg" class="img-responsive" alt="camera" />
+          </a>
+          <a href="/ProductListing/productListing.html" class="category__card">
+            <img src="/Assets/lens.jpg" class="img-responsive" alt="lens" />
             <span class="categogry__name txt-center txt-bold txt-2xl"
               >Lens</span
             >
-          </div>
-          <div class="category__card">
+          </a>
+          <a href="/ProductListing/productListing.html" class="category__card">
             <img
               src="/Assets/Accessories.jpg"
               class="img-responsive"
-              alt="camera"
+              alt="accessories"
               loading="lazy"
             />
             <span class="categogry__name txt-center txt-bold txt-2xl"
               >Accessories</span
             >
```

Comment:
beautify this part.... doesn't look good while reading...

Reply:
> beautify this part.... doesn't look good while reading...

I'm already using **Prettier**, it is rendering a few lines in this way only.